pkg/extparser/extparser.go

```diff
@@ -95,6 +99,12 @@ func (e *Ext2Parser) parseFileLine(line string) fsparser.FileInfo {
 	fi.Gid, _ = strconv.Atoi(res[0][4])
 	fi.Name = res[0][8]
 
+	if fi.IsLink() && strings.Contains(fi.Name, "->") {
```

Comment:
just incase, the `Contains` should be on the same string that you are splitting on (`" -> "` instead of `"->"`), otherwise if we somehow got `"->" in a string without spaces in it, the part below would apnic.